package.json

```diff
@@ -41,6 +41,8 @@
     "build:full": "npm run clean:full && npm run bootstrap && npm run build && npm run mocha && npm run lint",
     "pretest": "npm run clean && npm run build:current",
     "test": "node packages/build/bin/run-nyc npm run mocha",
+    "test:ci": "node packages/build/bin/run-nyc npm run mocha",
+    "posttest:ci": "npm run posttest",
```

Comment:
Please note `posttest:ci` is not referenced elsewhere. IIUC, the fix should be:

```
"test:ci": "node packages/build/bin/run-nyc npm run mocha && npm run posttest",
```

Reply:
`pretest` is not run by `test:ci` in your PR and it fails the build as no compile is done.

Reply:
> pretest is not run by test:ci in your PR and it fails the build as no compile is done.

I take it back as the `npm run bootstrap` triggers `prepare` scripts for all packages.

Reply:
Yep. Which in turn triggers a complete build (dist / dist6). The pretest cleans and re-builds the current target again.